packages/flutter_tools/test/integration.shard/test_data/project_with_immediate_exit.dart

```diff
@@ -0,0 +1,29 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'project.dart';
+
+class ProjectWithImmediateExit extends Project {
+
+  @override
+  final String pubspec = '''
+  name: test
+  environment:
+    sdk: ">=2.0.0-dev.68.0 <3.0.0"
+
+  dependencies:
+    flutter:
+      sdk: flutter
+  ''';
+
+  @override
+  final String main = r'''
+  import 'dart:io';
+
+  void main() {
```

Comment:
Thinking a bit more (and looking at the timeout) this could be flaky if it goes down too fast. Maybe an await for a few seconds?

Reply:
Added a 50ms delay. Not sure if things happening too fast or too slow caused the timeout, but if it turns out the test is just inherently flaky we might want to abandon it.